TRB_label_assist/clustering.py

```diff
@@ -130,6 +131,7 @@ def add_loc_clusters(
                 "metric": "od_similarity",
                 "similarity_threshold_meters": r,  # meters,
                 "apply_cutoff": False,
+                "clustering_way": clustering_way,
```

Comment:
Some additional context in the commit message or as a comment would be helpful.
Does the existing model already support `clustering_way`, for example?
What are the valid options?

`commit -m` is useful when you are learning git, not so much for long-term projects

Reply:
Yeah sure. I was going to write that after I raised https://github.com/e-mission/e-mission-server/pull/933 which supports this.

Reply:
>Does the existing model already support `clustering_way`, for example? What are the valid options?

No This config parameter is introduced in this PR. 
The valid options are :
`origin`
`destination`
`origin-destination`.

Previously, the entire model was using `origin-destination` as the default and only option for similarity.